pinot-segment-local/src/main/java/org/apache/pinot/segment/local/segment/index/readers/forward/VarByteChunkSVForwardIndexReader.java

```diff
@@ -103,11 +103,20 @@ private String getStringUncompressed(int docId) {
     long valueEndOffset = getValueEndOffset(chunkId, chunkRowId, chunkStartOffset);
 
     int length = (int) (valueEndOffset - valueStartOffset);
-    byte[] bytes = _reusableBytes.get();
+    byte[] bytes = getOrExpandByteArray();
     _dataBuffer.copyTo(valueStartOffset, bytes, 0, length);
     return new String(bytes, 0, length, UTF_8);
   }
 
+  private byte[] getOrExpandByteArray() {
+    byte[] bytes = _reusableBytes.get();
+    if (bytes == null || bytes.length < _lengthOfLongestEntry) {
```

Comment:
When can bytes be null? Or changing the question: If we assume bytes can be null, why do we initialize the ThreadLocal with `new byte[0]` instead of `null`?

Reply:
just a safety guard. I can remove the null